docs/UserGuide.md

```diff
@@ -111,25 +114,38 @@ Format: `edit INDEX [n/PROPERTY_NAME] [a/PROPERTY_ADDRESS] [s/SELLER_NAME] [p/SE
 Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st property to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Blk 298 Toa Payoh Central t/` Edits the name of the 2nd property to be `Blk 298 Toa Payoh Central` and clears all existing tags.
+* `edit 1 ta/4rm ta/near mrt` Edits the tag list of the 1st property by adding two tags called "4rm" and "near mrt" if they are not already present in the original tag list.
+* `edit 1 ta/4rm td/near mrt` Edits the tag list of the 1st property by adding a tag called "4rm" if it does not already exist in the original tag list and removing a tag called "near mrt" if it is present in the original tag list.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+The following example is invalid:
+
+`edit 1 t/near school ta/4rm td/near mrt` You cannot reset the tag list of a property, in this case to ["near school"], *and* modify the resetted tag list by adding a tag called "4rm" and removing a tag called "near mrt". The rationale is that this may be potentially confusing.
+
+</div>
+
 
 ### Locating properties by name: `find`
 
-Finds properties whose names contain any of the given keywords.
+Finds properties whose names contain any of the given keywords and whose tag list contain all of the specified tags.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find KEYWORD [MORE_KEYWORDS] [t/TAG_TO_MATCH]…`
 
-* The search is case-insensitive. e.g `hillview` will match `Hillview`
+* The keyword search is case-insensitive. e.g `hillview` will match `Hillview`
 * The order of the keywords does not matter. e.g. `Hillview Rise` will match `Rise Hillview`
 * Only the property name is searched.
 * Only full words will be matched e.g. `Han` will not match `Hillview`
-* Properties matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hillview Rise` will return `Hillview Grove`, `Rise Rivervale`
+* The tag search is case-sensitive.
```

Comment:
Should tag search be case-sensitive? For example I think "4 Rm" doesn't isn't semantically different from "4 rm", same for "mrt" and "MRT".

Reply:
Good idea. I will change this to be case insensitive.